dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/HostAPIImpl.java

```diff
@@ -355,7 +357,15 @@ public List<Host> findAllFromDB(final User user, final boolean respectFrontendRo
     @Override
     public List<Host> findAllFromDB(final User user, final boolean includeSystemHost,
                                     final boolean respectFrontendRoles) throws DotDataException, DotSecurityException {
-        return this.findPaginatedSitesFromDB(user, 0, 0, null, includeSystemHost, respectFrontendRoles);
+        return this.findAllFromDB(user, includeSystemHost, false, respectFrontendRoles);
+    }
+
+    @Override
+    public List<Host> findAllFromDB(final User user, final boolean includeSystemHost,
```

Comment:
I think  this method is calling a method which is not closing the conn, consider check if the method are conn safe?

Reply:
Added CloseDBIfOpened annotation to closing the connection if required